purchases/src/androidTest/kotlin/com/revenuecat/purchases/MainActivity.kt

```diff
@@ -2,9 +2,9 @@ package com.revenuecat.purchases
 
 import android.os.Bundle
 import android.widget.LinearLayout
-import androidx.appcompat.app.AppCompatActivity
+import androidx.fragment.app.FragmentActivity
 
-class MainActivity : AppCompatActivity() {
+class MainActivity : FragmentActivity() {
```

Comment:
I was running into a weird issue of a missing function when using `AppCompatActivity`... I tried to research a bit and didn't find anything. Going to the superclass fixed things, but we probably want to take another look when we have more time.

Reply:
Is it maybe due to the differences in the AndroidManifest?